uniffi_bindgen/src/interface/record.rs

```diff
@@ -141,6 +137,12 @@ impl Field {
     }
 }
 
+impl AsType for Field {
+    fn as_type(&self) -> Type {
+        self.type_.clone()
+    }
+}
```

Comment:
I'll just mention that another way to do this would be `impl Into<Type> for &Field`, but that doesn't feel like it's in the spirit of `Into` to me.  I like a separate trait better.